lib/resty/cookie.lua

```diff
@@ -91,7 +94,10 @@ local function get_cookie_table(text_cookie)
     end
 
     if key ~= nil and value == nil then
-        cookie_table[key] = sub(text_cookie, i)
+        if not cookie_table[key] then
+            cookie_table[key] = {}
+        end
+        table.insert(cookie_table[key], sub(text_cookie, i))
```

Comment:
We might also consider filling `last_value_cookie_table` here, since it is already being saved in the object properties in the `get_all` function?